oeps/processes/oep-0060-proc-sec-group.rst

```diff
@@ -51,52 +51,96 @@ Reports come into security@openedx.org
+
+.. _Common Vulnerability Scoring System (CVSS), version 3.1: https://www.first.org/cvss/v3.1/specification-document
+
+Maintainers are ultimately responsible for the security of their repositories
+-----------------------------------------------------------------------------
+Maintainers are responsible for resolving disclosed security vulnerabilies and notifying the security working group once they are resolved.
+
+To assist the maintainer with keeping track of their repository's vulnerabilities, the security working group will create a `GitHub repository security advisory`_ pre-assigned with the scored severity.
+
+Maintainers are requested to notify the security working group if they would like to adjudicate a vulnerability to a different severity than what was originally triaged. The security working group must accept the maintainer's adjudication, but should comment on any considerations around the adjudication.
+
+Maintainers will be reminded to remediate disclosures in proportion to the severity of the disclosure. The following table shows the reminder frequency until resolution of each severity classification:
+
+========    =====   ==================
+Severity    Score   Reminder frequency
+========    =====   ==================
+Low         ≥0.1    Once a semester
+Medium      ≥4.0    Once a quarter
+High        ≥7.0    Once a month
+Critical    ≥9.0    Once a week
+========    =====   ==================
+
+The on-call security working group member is responsible for sending reminders for any vulnerability reminders due during the time they are on-call.
 
 .. note::
 
-   We would also like to specifically note that if an on-call week is particularly time intensive, the on-call member can
-   ask the other working group members for support.
+    Maintainers are encouraged to reach out to security@openedx.org if they need any help remediating security issues.
+
+.. _GitHub repository security advisory: https://docs.github.com/en/code-security/security-advisories/repository-security-advisories/creating-a-repository-security-advisory
+
+Details of security vulnerabilities are private until a fix is released
+-----------------------------------------------------------------------
+Members of the Open edX community with access to the details of a disclosure are asked to keep those details private until a fix is publically released for the vulnerability.
+
+Maintainers should use GitHub's `temporary private fork`_ feature within the GitHub repository security advisory created by the security working group to keep the implementation details of a fix for a vulnerability private.
+
+.. _temporary private fork: https://docs.github.com/en/code-security/security-advisories/repository-security-advisories/collaborating-in-a-temporary-private-fork-to-resolve-a-repository-security-vulnerability
+
+Operators are ultimately responsible for the security of their Open edX instance
```

Comment:
Any reason this shouldn't just go into the `Guidance for operators` section?

Reply:
You're right, most of the information is down there already. Removed.